### PR TITLE
Vz api

### PR DESCRIPTION
**Pull request by jovenc**
## What this pull request accomplishes:

- Using Google Sheets API to ingest data from the Google Sheet maintained by the Vision Zero team namely 2018 and 2019 school safety zones, put them into postgres tables and automate the process using airflow. 

## Issue(s) this solves:

- #224 

## What, in particular, needs to reviewed:

- Added a vision_zero folder which contains a README.md and the Python scripts used to run the API
- Updated .gitignore to include credentials file to read Google Sheets
- Added a dag folder which contains the Airflow script used to run Airflow using Python Operator
- Edited the main README.md for bdit_data-sources

### CODE REVIEW

**Comment by radumas** [comment]
Question about two of the CodeFactor issues, do those extended comments you've added in triple double-quotes get parsed by Sphinx?

**Comment by jovenc** [comment]
If those two aren't working then I think there are the other two of the similar case that shouldn't work too. Only two are working as far as I can tell.

File: vision_zero/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Vision Zero Google Sheets API 
+This repository contains scripts to read Vision Zero google spreadsheets and put them into two postgres tables using Google Sheets API. This process is then automated using Airflow for it to run daily.
```
Comment (radumas) [inline vision_zero/README.md]: This folder

File: vision_zero/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    "private_key":
+    "client_email":
+    "client_id": 
+    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+    "token_uri": "https://oauth2.googleapis.com/token",
+    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+    "client_x509_cert_url": 
+  
+
+**Note:** A file `token.pickle` will be created if not found from the same directory. If the `SCOPES` is modified from readonly, the existing file `token.pickle` has to be deleted first.
```
[the remark this answers is not in the document]
Comment (jovenc) [reply vision_zero/README.md]: You're right. It is not created if we are using service_account.

**Comment by jovenc** [comment]
Made changes as per request

**Pull request merged**